core/modules/wsched/ChunkDisk.cc

```diff
@@ -139,6 +139,17 @@ bool ChunkDisk::ready(bool useFlexibleLock) {
 /// Precondition: _queueMutex must be locked
 /// Return true if this disk is ready to provide a Task from its queue.
 bool ChunkDisk::_ready(bool useFlexibleLock) {
+    auto logMemManRes =
+        [this](bool newVal, std::string const& msg, std::vector<memman::TableInfo> const& tblVect) {
+        if (setResourceStarved(newVal)) {
+            std::string str;
+            for (auto const& tblInfo:tblVect) {
+                str += tblInfo.tableName + " ";
+            }
+            LOGS(_log, LOG_LVL_DEBUG, "ready memMan " << msg << " - " << str);
```

Comment:
It would be nice. There is currently no << operator for TableInfo and I'm not sure where to put it as memman.h defines an interface and intentionally has no implementation. Maybe AndyH has something to say about it.